next/components/logo-strip.tsx

```diff
@@ -0,0 +1,47 @@
+import { useTranslation } from "next-i18next";
+import { ReactElement } from "react";
+
+import Facebook from "@/styles/icons/facebook.svg";
+import LinkedIn from "@/styles/icons/linkedin.svg";
+import Twitter from "@/styles/icons/twitter.svg";
+
+interface Logo {
+  image: ReactElement;
+  label: string;
+  id: string;
+}
+
+const logos: Logo[] = [
+  {
+    image: <Facebook className="inline-block h-16 w-16 text-primary-600" />,
+    label: "Share to Facebook",
```

Comment:
These labels would need translating

Reply:
Good catch, I actually copied this code from the social share branch to begin with and now I reckon we won't need to translate these as it will say simply "CompanyX logo" for now.